packages/pulseaudio.rb

```diff
@@ -38,15 +25,12 @@ class Pulseaudio < Package
   depends_on 'tdb'
 
   def self.build
-    system './configure',
-           "--prefix=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}"
+    system "./configure #{CREW_OPTIONS}"
     system 'make'
   end
 
   def self.install
-    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
-    FileUtils.mv "#{CREW_DEST_DIR}/lib", "#{CREW_DEST_PREFIX}/libexec"
+    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install' "-j#{CREW_NPROC}"
```

Comment:
Although not needed altogether, the `-j#{CREW_NPROC}` option should go in the `def self.build` function instead.
```
system 'make', "-j#{CREW_NPROC}"
```